SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -422,7 +422,14 @@ void UGlobalStateManager::HandleActorBasedOnLoadBalancer(AActor* Actor) const
 
 	if (!Actor->GetIsReplicated())
 	{
-		return;
+		if (USpatialStatics::IsSpatialOffloadingEnabled(Actor->GetWorld()) && !USpatialStatics::IsActorGroupOwnerForActor(Actor))
+		{
+			// We want to handle this non-replicated actor based on the load balancer, to support legacy offloading behavior.
```

Comment:
felt it was clearer this way, rather than inverting the condition, but this is also kind of weird.

Reply:
removed now